dev/checkstyle.xml

```diff
@@ -157,7 +157,8 @@
             <property name="allowSamelineMultipleAnnotations" value="true"/>
         </module>
         <module name="MethodName">
-            <property name="format" value="^[a-z][a-z0-9][a-zA-Z0-9_]*$"/>
+            <!-- Exclude Append and Complete were added in SPARK-15517 -->
+            <property name="format" value="^[a-z][a-z0-9][a-zA-Z0-9_]*$|Append|Complete"/>
```

Comment:
Hi, @techaddict .
Please use `dev/checkstyle-suppressions.xml` instead of changing rule.